src/workerd/io/worker-entrypoint.c++

```diff
@@ -753,10 +753,10 @@ kj::Promise<WorkerInterface::CustomEvent::Result> WorkerEntrypoint::customEvent(
 }
 
 #ifdef KJ_DEBUG
-void requestGc(const Worker& worker) {
+void requestGc(kj::Arc<Worker> worker) {
   TRACE_EVENT("workerd", "Debug: requestGc()");
-  jsg::runInV8Stack([&](jsg::V8StackScope& stackScope) {
-    auto& isolate = worker.getIsolate();
+  jsg::runInV8Stack([worker = kj::mv(worker)](jsg::V8StackScope& stackScope) {
```

Comment:
Nitpick: This runs synchronously so the move is redundant, I would stick with the `[&]` capture.